modules/ztp-sno-du-enabling-kdump.adoc

```diff
@@ -6,6 +6,28 @@
 [id="ztp-sno-du-enabling-kdump_{context}"]
 = Automatic kernel crash dumps with kdump
 
-`kdump` is a Linux kernel feature that creates a kernel crash dump when the kernel crashes. `kdump` is enabled with the following `MachineConfig` CR:
+`kdump` is a Linux kernel feature that creates a kernel crash dump when the kernel crashes. `kdump` is enabled with the following `MachineConfig` CRs.
```

Comment:
https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization

> Do not start a sentence with a lowercase term; rewrite the sentence.

Reply:
This is how it has been published (also peer reviewed) in previous releases: https://docs.openshift.com/container-platform/4.14/scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.html#ztp-sno-du-enabling-kdump_sno-configure-for-vdu

Suggestion to adhere to this rule:
> The `kdump` Linux kernel feature creates a kernel crash dump when the kernel crashes.  The `kdump` feature is enabled with the following `MachineConfig` CRs.